dev-java/jspeex/jspeex-0.9.7-r1.ebuild

```diff
@@ -1,43 +1,47 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="5"
+EAPI=7
 
 JAVA_PKG_IUSE="doc source test"
 
-inherit base java-pkg-2 java-ant-2
+inherit java-pkg-2 java-ant-2
 
 DESCRIPTION="Speex speech codec library for Java"
 HOMEPAGE="http://jspeex.sourceforge.net/"
 SRC_URI="mirror://sourceforge/${PN}/${P}.zip"
+S="${WORKDIR}/${PN}"
 
 LICENSE="BSD"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 
 CDEPEND="dev-java/ant-core:0"
-
-RDEPEND="${CDEPEND}
-	>=virtual/jre-1.4"
-
-DEPEND="${CDEPEND}
-	>=virtual/jdk-1.4
+RDEPEND="
+	${CDEPEND}
+	>=virtual/jre-1.4
```

Comment:
should be changed to `>=virtual/jre-1.8:*` and done in a revbump instead. though not related to eapi 7, it's a part of the ebuild cleanup :-) java 1.4 is not supported by java 11, in fact the lowest supported version by java 11 is 1.6 but we put there the lowest version supported on gentoo which is 1.8.

Reply:
Excellent point, thank you @fordfrog for your comments!